tests/test_web/test_api_v1.py

```diff
@@ -1316,3 +1316,149 @@ def test_regenerate_bundle_batch_invalid_input(payload, error_msg, app, auth_env
+        'build_requests': [
+            {
+                'bundles': ['registry-proxy/rh-osbs/lgallett-bundle:v1.0-9'],
+                'binary_image': 'registry-proxy/rh-osbs/openshift-ose-operator-registry:v4.5',
+                'from_index': 'registry-proxy/rh-osbs-stage/iib:v4.5',
+                'add_arches': ['amd64'],
+                'cnr_token': 'no_tom_brady_anymore',
+                'organization': 'hello-operator',
+                'overwrite_from_index': True,
+                'overwrite_from_index_token': 'some_token',
+            },
+            {
+                'operators': ['kiali-ossm'],
+                'binary_image': 'registry-proxy/rh-osbs/openshift-ose-operator-registry:v4.5',
+                'from_index': 'registry:8443/iib-build:11',
+            },
+        ],
+    }
+    rv = client.post('/api/v1/builds/add-rm-batch', json=data, environ_base=auth_env)
+
+    assert rv.status_code == 201, rv.json
+    assert mock_hrr.apply_async.call_count == 1
+    assert mock_har.apply_async.call_count == 1
+    mock_har.apply_async.assert_has_calls(
+        (
+            mock.call(
+                args=[
+                    ['registry-proxy/rh-osbs/lgallett-bundle:v1.0-9'],
+                    'registry-proxy/rh-osbs/openshift-ose-operator-registry:v4.5',
+                    1,
+                    'registry-proxy/rh-osbs-stage/iib:v4.5',
+                    ['amd64'],
+                    'no_tom_brady_anymore',
+                    'hello-operator',
+                    None,
+                    True,
+                    'some_token',
+                    None,
+                ],
+                argsrepr=(
+                    "[['registry-proxy/rh-osbs/lgallett-bundle:v1.0-9'], "
+                    "'registry-proxy/rh-osbs/openshift-ose-operator-registry:v4.5', 1, "
+                    "'registry-proxy/rh-osbs-stage/iib:v4.5', ['amd64'], '*****', "
+                    "'hello-operator', None, True, '*****', None]"
+                ),
+                link_error=mock.ANY,
+                queue=None,
+            ),
+        )
+    )
+    mock_hrr.apply_async.assert_has_calls(
+        (
+            mock.call(
+                args=[
+                    ['kiali-ossm'],
+                    'registry-proxy/rh-osbs/openshift-ose-operator-registry:v4.5',
+                    2,
+                    'registry:8443/iib-build:11',
+                    None,
+                    None,
+                    None,
+                ],
+                argsrepr=(
+                    "[['kiali-ossm'], 'registry-proxy/rh-osbs/openshift-ose-operator-registry:v4.5'"
+                    ", 2, 'registry:8443/iib-build:11', None, None, None]"
+                ),
+                link_error=mock.ANY,
+                queue=None,
+            ),
+        )
+    )
+
+    assert db.session.query(RequestAdd).filter_by(id=1).scalar()
+    assert db.session.query(RequestRm).filter_by(id=2).scalar()
```

Comment:
@yashvardhannanavati not a big deal, but the proper way to do this is as follows:
```python
assert db.session.query(RequestAdd).get(1)
```